caster-editor/src/components/Graph.vue

```diff
@@ -68,7 +68,8 @@ const graphPan = (location: graphPanType, event?: MouseEvent) => {
 
   switch (location) {
     case graphPanType.NodeEditor:
-      const editorWidth = document.getElementsByClassName('node-editor')[0]?.clientWidth || 0;
+      // const editorWidth = document.getElementsByClassName('node-editor')[0]?.clientWidth || 0;
+      const editorWidth = 800; // this needs to be hard coded for transition purposes
```

Comment:
how do you know its 800px? Why don't we calculate it on the fly?

Reply:
The problem is the vue `transition` property. This is what happens right now:

1. `ShowNodeEditor` toggles to true
2. Vue `Transition` animates the scriptcell panel from off screen into the screen from the side
3. The function for animating the graph node animation runs
4. Since the scriptcell panel animation has some delay (which seamed to be more than 1 tick), it's not rendered in the dom yet, so we don't know the width to use in `3`. -> we would need a longer timeout, before we can start the animation of the nodes.

Considering our mid-term goal of replacing `v-network-graph` I chose to hardcode the width. Since this function will be obsolete once we replace it.

The options are:

1. Build a custom animation in and out with gsap and have the scriptcell editor always rendered off or on screen. -> Then we can always acess its true size (this would probably be the future proof option).
2. Add a hardcoded delay to the node animation
3. Export the scriptcell width as a variable and use that within the js